src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/DocumentDbOperations.java

```diff
@@ -18,72 +18,344 @@
 import java.util.List;
 import java.util.Optional;
 
+/**
+ * provide query operations for
+ * ${@link com.microsoft.azure.spring.data.cosmosdb.repository.support.SimpleDocumentDbRepository}
+ *
+ * <p>
+ * <b>Collection</b>
+ * A collection is a container of JSON documents and the associated JavaScript application logic. Collections can span
```

Comment:
What is the `JavaScript` application logic here?

Reply:
no, just html label, learned from RxJava doc.